test/verify.test.js

```diff
@@ -567,7 +567,7 @@ test.serial("Throw error if GitLab API return any other errors", async (t) => {
   const owner = "test_user";
   const repo = "test_repo";
   const env = { GITLAB_TOKEN: "gitlab_token" };
-  const gitlab = authenticate(env).get(`/projects/${owner}%2F${repo}`).times(3).reply(500);
+  const gitlab = authenticate(env).get(`/projects/${owner}%2F${repo}`).times(3).reply(500).persist();
```

Comment:
.persist is needed to avoid confusing nock errors.

Reply:
See [above](https://github.com/semantic-release/gitlab/pull/728/files#r1627183757)